packages/core/package.json

```diff
@@ -72,6 +73,8 @@
     "prettier": "^3.0.3",
     "prettier-plugin-svelte": "^3.0.3",
     "prettier-plugin-tailwindcss": "^0.5.4",
+    "prism-themes": "^1.9.0",
+    "prismjs": "^1.29.0",
```

Comment:
Can someone remind me the difference between adding these to `dependencies` vs `devDependencies` in `svelte-package` world? My instinct would be that they need to be in `dependencies`, like other libs we import directly in components

Reply:
Oh good catch. We had them as `devDependencies` because we were dynamically loading everything from the CDN. Now that we don't do that, we should probably move `prismjs` and `prism-themes` to dependencies now.